deps/build.jl

```diff
@@ -0,0 +1,33 @@
+if !haskey(ENV, "MKLROOT")
+    error("please set the environmental variable MKLROOT to build MKLSparse.jl")
+end
+
+if Base.USE_BLAS64
+    libmkl_path = joinpath(ENV["MKLROOT"], "lib", "intel64", "libmkl_rt")
```

Comment:
why wouldn't ccall'ing to `:libmkl_rt` and letting normal library path searching and resolution do its thing work here?